gulpfile.js

```diff
@@ -151,158 +97,111 @@ gulp.task('clean-dev', function() {
         './dist/*.png',
         './dist/*.ico',
         './reports/**/*',
-        './reports']);
+        './reports'
+    ]);
 });
 
-gulp.task('clean-full', function() {
+gulp.task('clean-full', function () {
     del(['./dist/*',
         './reports/**/*',
-        './reports']);
+        './reports'
+    ]);
 });
 
 
 // =======================================================================
 // JSHint
 // =======================================================================
-gulp.task('lint', function() {
+gulp.task('lint', function () {
     return gulp.src(filePath.lint.src)
-    .pipe(jshint())
-    .pipe(jshint.reporter(stylish));
+        .pipe(jshint())
+        .pipe(jshint.reporter(stylish));
 });
 
 
 // =======================================================================
 // Javascript Checkstyles (JSCS)
 // =======================================================================
-gulp.task('checkstyle', function() {
+gulp.task('checkstyle', function () {
     return gulp.src(filePath.lint.src)
-    .pipe(jscs())
-    .on('error', handleError);
+        .pipe(jscs())
+        .on('error', handleError);
 });
 
 
 // =======================================================================
 // Browserify Bundle
 // =======================================================================
 
-// var bundle = {};
-// bundle.conf = {
-//     entries: filePath.browserify.src,
-//     external: filePath.vendorJS.src,
-//     debug: true,
-//     cache: {},
-//     packageCache: {}
-// };
-
-// function configureBundle(prod) {
-//     var bundler = browserify(bundle.conf);
-//
-//     if (!prod) {
-//         bundler = watchify(bundler);
-//         bundler.on('update', rebundle);
-//     }
-//
-//     bundler.transform(envify);
-//
-//     bundle.bundler = bundler;
-//     bundle.prod = prod;
-// }
-
-gulp.task('bundle-dev', function () {
-    // 'use strict';
-    // configureBundle(false);
-    // return rebundle();
-});
-
-gulp.task('bundle-prod', function () {
-    // 'use strict';
-    // configureBundle(true);
-    // return rebundle();
+gulp.task('bundle', function () {
+    return browserify({
+            entries: filePath.browserify.src,
+            debug: true
+        })
+        .transform(browserifyCss, {
+            global: true,
+        })
+        .transform(markedify)
+        .bundle() // Start bundle
+        .pipe(source(filePath.browserify.src)) // Entry point
+        .pipe(buffer()) // Convert to gulp pipeline
+        .pipe(sourceMaps.init({
+            loadMaps: true
+        })) // Strip inline source maps
+            .pipe(ngAnnotate())
+            .pipe(uglify({
+                mangle: false
```

Comment:
> d3: 237kb (are we using this anywhere?)

Long ago some folks were playing with visualizations but nothing is implemented, we can remove this.

> angular-ui-bootstrap: 171kb

I propose we remove this, I think the plan was to use more components from ui-bootstrap but I don't think it's been really worth it. Just my 2¢

> jquery: 129kb (?)

This is a dependency for some of the bootstrap/ui-bootstrap components. I think we're at least using the nav on mobile which requires this, but I would also be happy to come up with a bespoke solution if it means dropping ui-bootstrap and jquery.

> lodash: 124kb

I'd also be happy to drop this. IIRC, theres a handful of array iteration functions we're using that can just be replaced with an ES5/ES6 shim.

> swagger-client: 70kb (should be commented out at least)

Yep, I feel like we won't return to it anytime soon. I'd like to remove this as part of a PR removing `disclosureApi`. Happy to take that on unless you want it.